### PR TITLE
charts: merge synapse down into matrix-stack

### PR DESCRIPTION
**Pull request by gaelgatelement**
- Follow up of https://github.com/element-hq/ess-helm/pull/41 - Synapse is now part of matrix-stack chart

### CODE REVIEW

File: REUSE.toml [hunk 1]
```diff
@@ -15,6 +15,11 @@ path = "charts/**/source/sub_schemas/*.json"
 SPDX-FileCopyrightText = "Copyright 2024 New Vector Ltd"
 SPDX-License-Identifier = "AGPL-3.0-only OR LicenseRef-Element-Commercial"
 
+[[annotations]]
+path = "charts/**/sub_schemas/*.json"
```
Comment (benbz) [inline REUSE.toml:14]: Unsure if the component specific sub-schemas should go directly in `source/<component>` rather than `source/<component>/sub_schemas`. And possibly the common ones in `source/common` or `source/library` so everything is consistent. That can be a separate PR though

Regardless I think the annotation above disappears

Comment (gaelgatelement) [reply REUSE.toml:14]: I like your reorg suggestion. I'll do so in a follow-up PR.
